### PR TITLE
Fix some SPM issues

### PR DESCRIPTION
**Pull request by gbrixey**
This is to fix an SPM issue in Xcode 12.5. It seems SPM doesn't like it when there is a directory at the same level as the umbrella header.
```
target 'SVProgressHUD' has invalid header layout: umbrella header found at '/Users/glenb/Code/SVProgressHUD/SVProgressHUD/SVProgressHUD.h', but directories exist next to it: /Users/glenb/Code/SVProgressHUD/SVProgressHUD/Images.xcassets; consider removing them
```

Also, for some reason the `SWIFTPM_MODULE_BUNDLE` macro was not available, so I added a Swift target in order to expose `Bundle.module` to the ObjC code a different way. Hopefully that will work.

### CODE REVIEW

File: SVProgressHUD/SVIndefiniteAnimatedView.m [hunk 1]
```diff
@@ -60,7 +61,7 @@ - (CAShapeLayer*)indefiniteAnimatedLayer {
         
         CALayer *maskLayer = [CALayer layer];
 
-        NSBundle *imageBundle = SWIFTPM_MODULE_BUNDLE;
```
Comment (veeamd) [inline SVProgressHUD/SVIndefiniteAnimatedView.m:63]: If I remember correctly this macro will be generated only when there're resources in that bundle.

**Review by veeamd — COMMENTED** [review]
Actually SVProgressHUD.h is not an umbrella header. Would you try add an `include` umbrella directory, and symlink all the headers into that folder and keep the image assets where it was?

**Comment by gbrixey** [comment]
> Would you try add an include umbrella directory, and symlink all the headers into that folder and keep the image assets where it was?

I tried to do this, but I couldn't get it to work. I tried something different, renaming `SVProgressHUD` so that Xcode doesn't think it's supposed to be an umbrella header. https://github.com/StreetEasy/SVProgressHUD/pull/3

File: SVProgressHUD/SVIndefiniteAnimatedView.m (hunk 1 above)
Comment (gbrixey) [reply SVProgressHUD/SVIndefiniteAnimatedView.m:63]: Yes, it looks like moving the image assets folder up one level made them inaccessible. I think the other PR #3 is a better fix.

**Comment by gbrixey** [comment]
I'll close this PR since I think #3 is a better fix.

**Pull request closed without merging**